schema.graphql

```diff
@@ -98,34 +110,19 @@ type TokenWeight @entity {
 }
 
 enum EpochState {
-    "active"
-    "transitioning"
-    "expired"
+    active
+    transitioning
+    expired
 }
 
 type System @entity {
     "pricing and constant"
-    """
```

Comment:
seem like it doesn't support multi-line comments

Reply:
OK. Is there any other way to add the comments?